format.js

```diff
@@ -173,6 +173,20 @@ function makeEncryptionFormat() {
     })
   }
 
+  function getGroupInfoUpdates(groupId) {
+    if (!groupId) return pull.error(new Error('Group id required'))
+
+    return pull(
+      pull.values([0]),
+      pull.asyncMap((_, cb) => {
+        keyringReady.onReady(() => {
+          return cb(null, keyring.group.getUpdates(groupId))
+        })
+      }),
+      pull.flatten()
```

Comment:
honestly this dummy => asyncMap into stream => flatten   ... I don't know how calling back with a stream works with flattern, but it does and I feel uncomfortable.

Fine, or use `pull-defer` for less dark-art solution?

Reply:
changed!